src/bundles.sh

```diff
@@ -0,0 +1,18 @@
+export CONDUCTR_IP=$(hostname -i)
+
+conduct load conductr-haproxy
+conduct run conductr-haproxy --scale $NODE_COUNT
+
+sudo service haproxy restart
+
+sleep 5
+
+ES_COUNT=$(( NODE_COUNT < 3 ? NODE_COUNT : 3 ))
+
+conduct load conductr-elasticsearch
+conduct run conductr-elasticsearch --scale $ES_COUNT
+
+sleep 5
```

Comment:
I recall an issue around race conditions between load and run, but that should now be solved. 

Perhaps better to poll via `docker ps' in order to wait for HAProxy, although conductr-haproxy will tolerate HAProxy not being there (if there's any problem in that regard then we should fix it).